pubspec.yaml

```diff
@@ -48,6 +49,10 @@ flutter:
   assets:
     - images/PairingsBg.png
     - images/DefaultBg.png
+    - images/Logo.png
+    
+  #  - images/a_dot_burr.jpeg
```

Comment:
Delete comments.

Reply:
I think it is a sample

Reply:
Yes, it is! Since we already are importing assets from line 50-52, we don't really need those comments anymore!